src/main/scala/molmed/utils/GeneralUtils.scala

```diff
@@ -303,6 +305,12 @@ class GeneralUtils(projectName: Option[String], uppmaxConfig: UppmaxConfig) exte
       else
         ""
 
+    def skiplDuplicated =
+      if(skipDup != -1)
+        "--skip-duplicated --skip-dup-mode " + skipDup
```

Comment:
I think that this should be fine. I think that the String object has implicit conversions.